src/Hng.Infrastructure/Services/SeederService.cs

```diff
@@ -144,6 +144,32 @@ public async Task SeedProducts()
 
     }
 
+    public async Task SeedJobListings()
+    {
```

Comment:
Your task is to make an endpoint for creating job listing, why are you seeding dummy job listing?
Take this out.